mcs/class/System/Test/System.Net.NetworkInformation/NetworkInterfaceTest.cs

```diff
@@ -70,6 +77,9 @@ public void FirstInterfaceOperationalStatus ()
 		[Test]
 		public void FirstInterfaceSpeed ()
 		{
+			if (RunningOnUnix)
+				Assert.Ignore ();
+
 			NetworkInterface[] adapters = NetworkInterface.GetAllNetworkInterfaces ();
 			Assert.IsTrue (adapters[0].Speed > 0);
```

Comment:
we seem to be just returning a hardcoded `1000000` in the Mono implementation while corefx always throws a NotImplementedException. I wonder how many apps that will break and if we need to do something about it?